OpenEdXMobile/res/layout/fragment_whats_new.xml

```diff
@@ -2,65 +2,74 @@
+            android:layout_height="wrap_content">
 
-        <com.joanzapata.iconify.widget.IconTextView
-            android:id="@+id/close_btn"
-            style="@style/semibold_text"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_gravity="top|right|end"
-            android:background="@drawable/selectable_box_overlay"
-            android:contentDescription="@string/label_close"
-            android:padding="@dimen/widget_margin"
-            android:text="{fa-close}"
-            android:textColor="@color/white"
-            android:textSize="@dimen/edx_x_large"
-            tools:targetApi="JELLY_BEAN_MR1" />
+            <TextView
+                android:id="@+id/screen_title"
+                style="@style/semibold_text"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center_horizontal"
+                android:layout_marginLeft="@dimen/edx_double_margin"
+                android:layout_marginRight="@dimen/edx_double_margin"
+                android:gravity="center"
+                android:padding="@dimen/widget_margin"
+                android:textColor="@color/edx_white"
+                android:textSize="@dimen/edx_base"
+                tools:text="Version 2.8.1 New Features" />
+
+            <com.joanzapata.iconify.widget.IconButton
+                android:id="@+id/close_btn"
+                style="@style/semibold_text"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="top|right|end"
+                android:background="@drawable/selectable_box_overlay"
+                android:contentDescription="@string/label_close"
+                android:padding="@dimen/widget_margin"
+                android:text="{fa-close}"
+                android:textColor="@color/white"
+                android:textSize="@dimen/edx_x_large"
+                tools:targetApi="JELLY_BEAN_MR1" />
+        </FrameLayout>
 
         <android.support.v4.view.ViewPager
             android:id="@+id/view_pager"
             android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:layout_marginBottom="@dimen/whats_new_pager_vertical_margin"
-            android:layout_marginTop="@dimen/whats_new_pager_vertical_margin" />
+            android:layout_height="0dp"
+            android:layout_weight="1" />
 
         <FrameLayout
-            android:id="@+id/indicator_container"
-            android:layout_width="wrap_content"
-            android:layout_height="@dimen/whats_new_pager_vertical_margin"
-            android:layout_gravity="bottom|center"
-            android:padding="@dimen/widget_margin" />
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/whats_new_bottom_panel_height">
```

Comment:
It should be `wrap_content`, we can set margin/padding of inner views if required.